support-workers/src/main/scala/com/gu/emailservices/DigitalPackEmailFields.scala

```diff
@@ -59,6 +61,12 @@ object DigitalSubscriptionEmailAttributes {
       default_payment_method: String = "Direct Debit",
     ) extends PaymentFieldsAttributes
 
+    case class SepaAttributes(
+      // TODO
```

Comment:
Is this TODO still necessary?

Reply:
Perhaps not. This and the other one in `PaperFieldsGenerator.scala` are subscriptions-specific code - we've updated them to make the pattern matching exhaustive but they will not be used for contributions.
I guess if we do add support for subs, we'll test these paths anyway, so it's safe to remove the TODOs